app/lib/mavis_cli/organisations/create_sessions.rb

```diff
@@ -24,6 +23,8 @@ def call(ods_code:, academic_year:)
           return
         end
 
+        academic_year = AcademicYear.pending if academic_year.nil?
```

Comment:
```suggestion
        academic_year ||= AcademicYear.pending
```

I think this looks slightly neater in my opinion.